src/kw_config_loader.sh

```diff
@@ -316,6 +316,11 @@ function parse_configuration()
       value="$(cut -d '=' -f 2- <<< "${line%#*}")"
       value="$(sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' <<< "$value")"
 
+      if [[ "$varname" == 'mail_groups' ]]; then
+        eval "${config_array}"'["$varname"]+="$value,"'
+        continue
+      fi
```

Comment:
Do we really need this behavior here? `kw_config_loader` has a well-defined behavior, and this code changes it a little bit; I would prefer a change like that inside the mail file.